runhouse/rns/folders/folder.py

```diff
@@ -744,10 +745,10 @@ def exists_in_system(self):
             self.fsspec_url
         ) or rh.rns.top_level_rns_fns.exists(self.path)
 
-    def delete_in_system(self):
-        """Delete all contents in folder from file system."""
+    def delete_in_system(self, path: Optional[str] = None):
```

Comment:
would just `delete` make sense as the function name?

Reply:
i was following the convention we used for other resources (blob, folder, table, etc.).. do you think we should rename all of them for simplicity?